src/main/scala/no/ndla/draftapi/service/search/SearchConverterService.scala

```diff
@@ -55,5 +61,47 @@ trait SearchConverterService {
       )
     }
 
+    def getHits(response: JestSearchResult, language: String): Seq[api.ArticleSummary] = {
```

Comment:
Er dette for å hente versjonen i Draft-API basert på søkeresultater fra Article-API?

Reply:
Dette er en funksjon som brukes i forbindelse med å konvertere et "hit" fra draft-api sin elasticsearch, og til en `ArticleSummary` som returneres fra søke-endepunktet :)

Reply:
Vet ikke om jeg egentlig skjønte det. Men høres fornuftig ut :)